node/node.go

```diff
@@ -609,6 +612,7 @@ waitPeer:
 			CertIssuerPublicKey: issuer.PublicKey,
 			CertIssuerSubject:   issuer.Subject,
 		},
+		FIPS: n.config.FIPS,
```

Comment:
While this doesn't really affect this PR, I'd like for us to make sure that users know to set this in the node config.

Reply:
@nishanttotla I agree - where would you suggest this be documented?

Reply:
@cyli I'm not sure, but let me look this up. My hunch is that this would go into the Docker API documentation. cc @thaJeztah

Reply:
Trying to grasp where / when it needs to be set; @cyli @nishanttotla feel free to ping me on Slack / drop by my desk to explain 😅